src/main/deltachat/login.ts

```diff
@@ -221,7 +221,6 @@ export function txCoreStrings() {
   strings[C.DC_STR_AUDIO] = tx('audio')
   strings[C.DC_STR_FILE] = tx('file')
   strings[C.DC_STR_ENCRYPTEDMSG] = tx('encrypted_message')
-  strings[C.DC_STR_STATUSLINE] = tx('pref_default_status_text')
```

Comment:
technically this doesn't belong in the pr as it is for making the code work with the newest core,
which is already done in #2530

Reply:
Thanks for reviewing! So now I should remove the changes from this file from the PR even if the tests fail then?

Reply:
The tests should only fail locally for you if you use a newer deltachat-node :)